src/schemas/Entity.js

```diff
@@ -63,7 +63,7 @@ export default class EntitySchema {
 
   denormalize(entityOrId, unvisit, getDenormalizedEntity) {
     const entity = getDenormalizedEntity(this, entityOrId);
-    if (typeof entity !== 'object') {
+    if (typeof entity !== 'object' || entity === null) {
```

Comment:
Discovered a fun fact about javascript while debugging this: `typeof null === object`